### ISSUE TITLE
Ruby version 2.3.1 released. Please update windows binary download

### ISSUE THREAD

**Issue by duytrung** [oneclick/rubyinstaller#318]
Hi,

Ruby version 2.3.1 released. Please update windows binary download. 

I'm eager to try newer version.

Great thanks!

**Comment by RichIsOnRails** [comment]
Note that the referenced pull request works fine.  I'm currently running:

`ruby 2.3.1p112 (2016-04-26 revision 54768) [i386-mingw32]`

**Comment by betam4x** [comment]
Same, outside the usual early adopter issues (nokogiri, I'm looking at you) things are working great.

**Comment by Azolo** [comment]
I looked at the current PR and I started building it but I'm on a not great connection at the moment and had problems downloading the dependencies from sourceforge.

Hopefully I'll have better luck when I get home, but I don't know when I'll get a chance to look at it either.

Until then here are some things that would make releases faster. =)
https://github.com/oneclick/rubyinstaller/issues/321#issuecomment-219296160
